macros/models/_sector/nft/nft_transfers.sql

```diff
@@ -0,0 +1,117 @@
+{% macro nft_transfers(blockchain, base_transactions, erc721_transfers, erc1155_single, erc1155_batch ) %}
+{%- set token_standard_721 = 'bep721' if blockchain == 'bnb' else 'erc721' -%}
+{%- set token_standard_1155 = 'bep1155' if blockchain == 'bnb' else 'erc1155' -%}
+{%- set spark_mode = True -%} {# TODO: Potential bug. Consider disabling #}
+SELECT
+    *
+FROM(
+     SELECT '{{blockchain}}' as blockchain
+    , t.evt_block_time AS block_time
+    , date_trunc('day', t.evt_block_time) AS block_date
+    , t.evt_block_number AS block_number
+    , '{{token_standard_721}}' AS token_standard
+    , 'single' AS transfer_type
+    , t.evt_index
+    , t.contract_address
+    , t.tokenId AS token_id
+    , cast(1 as uint256) AS amount
```

Comment:
minor, but this uses the less preferred casting technique?

Reply:
I think we do 1 big regex replace at the end of the migration for this.. 
We've not settled on what would be the preferred way until recently so a lot of early migrated models will have the other approach.